common/app/views/fragments/amp/stylesheets/tones.scala.html

```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```

Comment:
OK, Google says they're not enforcing this for the Preview. So nested selectors are OK - for now.